arithmetization/src/main/java/net/consensys/linea/zktracer/module/mmu/instructions/ModexpData.java

```diff
@@ -263,7 +263,9 @@ public MmuData setMicroInstructions(MmuData mmuData) {
       middleMicroInstruction(mmuData, sourceLimbOffset, targetLimbOffset);
     }
 
-    lastMicroInstruction(mmuData);
+    if (initialTotalNonTrivial > 1) {
```

Comment:
this is a fix

Reply:
⭐

Reply:
if the nb of non trivial mmio inst is 1, we just have a `only` mmio inst. We have a `last` mmio inst only if we have 2 or more non trivial inst. Before the fix, when we had only one non trivial mmio inst, we were writting one `only` mmio inst and then one `last` mmio inst.